.github/wheel-tests/test_tutorials.py

```diff
@@ -0,0 +1,102 @@
+import os
+import pathlib
+import signal
+import subprocess
+import sys
+
+import pytest
+import ROOT
+
+ROOT.gROOT.SetBatch(True)
+
+tutorial_dir = pathlib.Path(str(ROOT.gROOT.GetTutorialDir()))
+
+subdirs = ["analysis/dataframe", "analysis/tree", "hist", "io/ntuple", "roofit/roofit"]
+
+SKIP_TUTORIALS = {
+    "rf503_wspaceread.py",
+    "rf615_simulation_based_inference.py",
+    "rf618_mixture_models.py",
+    "ntpl004_dimuon.C",
+    "ntpl008_import.C",
+    "distrdf004_dask_lxbatch.py",
+}
```

Comment:
The tutorials we skip should also have some documentation about the reason they are skipped. Ideally we wouldn't skip anyone, but that is not always possible. By quickly looking at this list, I can tell that `distrdf004_dask_lxbatch.py` needs to be skipped always because it only works on lxplus. For the two RNTuple tutorials, I guess they are skipped because they require reading remote data via HTTP and our wheels don't enable that yet. For the others I'm not sure what's the reason for skipping yet.